script/deploy/deploySingleContract.sh

```
@@ -116,7 +122,12 @@ deploySingleContract() {
   fi
 
   # check if deploy script exists
-  local FULL_SCRIPT_PATH=""$DEPLOY_SCRIPT_DIRECTORY""$SCRIPT"".s.sol""
+  if [[ $NETWORK == "zksync" ]]; then
+    FULL_SCRIPT_PATH=""$DEPLOY_SCRIPT_DIRECTORY""$SCRIPT"".zksync.s.sol""
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)